cdip_admin/integrations/management/commands/fix_migrated_route_configs.py

```diff
@@ -0,0 +1,66 @@
+import copy
+
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from integrations.models import Integration
+
+
+DEFAULT_FIELD_MAPPING = {"default": "", "destination_field": "provider_key"}
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        def extract_provider_key(d):
+            for k, v in d.items():
+                if k == "default":
+                    return v
```

Comment:
I think this is OK for this narrow case to fix the Inreach RouteConfigurations.

Reply:
Yes, it was designed to work only in this case 😅  I know it is not the fanciest way but it works